integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -78,19 +78,19 @@ def _partition_items(self, channel_metadata_item_map):
             'Preparing to transmit creation of [%s] content metadata items with plugin configuration [%s]: [%s]',
             len(items_to_create),
             self.enterprise_configuration,
-            items_to_create.keys(),
+            list(items_to_create.keys()),
```

Comment:
?

Reply:
same behavior as of dict.values()

https://www.python.org/dev/peps/pep-3106/